docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md

```diff
@@ -107,6 +107,10 @@ For more information on plugin deprecation and how to request your plugin to be
 
 - No. We will not accept any new plugin submissions written in Angular. For more information, refer to our [Angular support deprecation documentation](https://grafana.com/docs/grafana/latest/developers/angular_deprecation/).
 
+### Can I submit plugins built with Toolkit?
+
+- The Toolkit is deprecated. Please [migrate to `create-plugin`](../migration-guides/migrate-from-toolkit.mdx). In the future, we will reject submissions based on Toolkit as it becomes increasingly out-of-date.
```

Comment:
I think for clarity's sake it would be best to use the package name `@grafana/toolkit` here.

Reply:
done.